packages/shopify-app-session-storage-drizzle/README.md

```diff
@@ -0,0 +1,41 @@
+# Session Storage Adapters for Drizzle
+
+This package implements the `SessionStorage` interface that works with an instance of [Drizzle](https://orm.drizzle.team).
+
+There are 3 adapters for Drizzle: `DrizzlePgAdapter`, `DrizzleSQLiteAdapter` and `DrizzleMySQLAdapter` which is in development. 
+
+Session storage for Drizzle requires a `schema.ts` with a `session` table. Make sure to create `session` table and apply changes to the database before using this pakacge.
+
+## Example with SQLite
+
+```ts
+import {sqliteTable, text, integer, blob} from 'drizzle-orm/sqlite-core';
+
+export const session = sqliteTable('session', {
```

Comment:
Nit: could we call this `sessionTable` instead? We usually use `session` in other places in the app and this might get a little confusing.